assignment-1/Players.java

```diff
@@ -0,0 +1,53 @@
+public class Players {
+    private String name;
+    private String team;
+    private String role;
+    private Integer matches;
+    private Integer runs;
+    private Double average;
+    private Double strikeRate;
+    private Integer wickets;
+    private Integer allRounders;
+    public Players() {}
+
+    public Players(String name, String team, String role, Integer matches, Integer runs, Double average, Double strikeRate, Integer wickets) {
+        this.name = name;
+        this.team = team;
+        this.role = role;
+        this.matches = matches;
+        this.runs = runs;
+        this.average = average;
+        this.strikeRate = strikeRate;
+        this.wickets = wickets;
+        this.allRounders = wickets*runs;
```

Comment:
this is only applicable if the role is an All-rounder. but since there is no distinction, this will be applied to both bowlers and batsmen.
it should have made sense to model your player class such that you have an AllrounderPlayer which extends from the Player class and implement this method only this the sub class.